include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -449,7 +449,7 @@ struct Value : Base<OrtValue> {
   static Value CreateTensor(const OrtMemoryInfo* info, void* p_data, size_t p_data_byte_count, const int64_t* shape, size_t shape_len,
                             ONNXTensorElementDataType type);
 
-
+#if !defined(DISABLE_SPARSE_TENSORS)
   /// <summary>
```

Comment:
For things that are part of the API it's probably better to do the ifdef inside the implementation and return an error if spare tensor support is disabled (use a helper function so the binary size cost of returning that message is minimized).